Packages/com.unity.inputsystem/InputSystem/Editor/ControlPicker/InputControlPathEditor.cs

```diff
@@ -89,10 +90,10 @@ public void OnGUI()
             EditorGUILayout.EndHorizontal();
         }
 
+        //TODO: on next major version remove property argument.
```

Comment:
I would recommend filing a JIRA ticket on this if/when this lands, otherwise it would be forgotten.